s03_configs/oav.yaml

```diff
@@ -0,0 +1,16 @@
+- tickit_devices.oav.oav_edge_detection.OAV_DI_OAV:
+    name: oav_di_oav
+    inputs: {}
+    port: 25565
+- tickit_devices.oav.oav_edge_detection.OAV_DI_IOC:
+    name: oav_di_ioc
+    inputs: {}
+    port: 25564
+- tickit_devices.oav.oav_edge_detection.OAV_EA_FSCN:
+    name: oav_ea_fscn
+    inputs: {}
+    port: 25563
+- tickit_devices.oav.oav_edge_detection.OAV_EA_OAV:
+    name: oav_ea_oav
+    inputs: {}
+    port: 25562
```

Comment:
Must: These need to be different ports to the other s03 devices so we can run them at the same time